src/components/ui/PaymentRequestDetailsModal/PaymentRequestDetailsModal.tsx

```diff
@@ -0,0 +1,92 @@
+import { Fragment } from 'react';
+import { Dialog, Transition } from '@headlessui/react';
+import PaymentRequestDetails from '../PaymentRequestDetails/PaymentRequestDetails';
+import { LocalPaymentRequest, LocalTag } from '../../../types/LocalTypes';
+
+export interface PaymentRequestDetailsModalProps {
+  paymentRequest: LocalPaymentRequest;
+  merchantTags: LocalTag[];
+  hostedPaymentLink: string;
+  onRefundClick: (paymentAttemptID: string) => void;
+  onTagAdded: (tag: LocalTag) => void;
+  onTagDeleted: (id: string) => void;
+  onTagCreated: (tag: LocalTag) => void;
+  open: boolean;
+  onDismiss: () => void;
+}
+
+const PaymentRequestDetailsModal = ({
+  paymentRequest,
+  merchantTags,
+  hostedPaymentLink,
+  onRefundClick,
+  onTagAdded,
+  onTagDeleted,
+  onTagCreated,
+  open,
+  onDismiss,
+}: PaymentRequestDetailsModalProps) => {
+  return (
+    <Transition appear show={open} as={Fragment}>
+      <Dialog as="div" onClose={onDismiss}>
+        <Transition.Child
+          as={Fragment}
+          enter="ease-in-out duration-300"
+          enterFrom="opacity-0"
+          enterTo="opacity-100"
+          leave="ease-in-out duration-300"
+          leaveFrom="opacity-100"
+          leaveTo="opacity-0"
+        >
+          <div className="fixed inset-0 bg-black bg-opacity-[15%]" />
+        </Transition.Child>
+        <div className="fixed inset-y-0 right-0">
+          <Transition.Child
+            as={Fragment}
+            enter="transform transition ease-in-out duration-300"
+            enterFrom="translate-x-full"
+            enterTo="translate-x-0"
+            leave="transform transition ease-in-out duration-300"
+            leaveFrom="translate-x-0"
+            leaveTo="translate-x-full"
+          >
+            <Dialog.Panel>
+              <div className="absolute left-0 top-0 -ml-8 flex pr-2 pt-4 sm:-ml-10 sm:pr-4">
+                <button type="button" className="opacity-0 focus:outline-none" onClick={onDismiss}>
```

Comment:
I'd recommend changing the `opacity-0` to `invisible` as we won't get the cursor-pointer when accidentally hovering over an invisible button

Reply:
invisible makes it hidden and the focus goes to the transactions table.